gossipsub.go

```diff
@@ -547,7 +555,7 @@ func (gs *GossipSubRouter) flush() {
 	}
 }
 
-func (gs *GossipSubRouter) pushGossip(p peer.ID, ihave *pb.ControlIHave) {
+func (gs *GossipSubRouter) queueGossip(p peer.ID, ihave *pb.ControlIHave) {
```

Comment:
Would be nice to have a comment saying that the queued gossip will be sent/piggy-backed with the next set of control messages or flushed during the next heartbeat.

Reply:
I might sound a little pedantic, but `queue` is not a verb; we enqueue, which is synonymous with pushing to a queue.

Reply:
It is pedantic, but it's also accurate! Changing.

Reply:
@aarshkshah1992

> Would be nice to have a comment saying that the queued gossip will be sent/piggy-backed with the next set of control messages or flushed during the next heartbeat.

That's part of the overall design of gossipsub; a one-off comment is not the appropriate place to convey that aspect to the user.